README.md

```diff
@@ -1,3 +1,10 @@
 # Using directives
 
-Library for processing meta-information written in using-directives syntax.
\ No newline at end of file
+Library for processing meta-information written in using-directives syntax.
+
+### Generating code coverage report
+Run:
+```shell
+./gradlew jacocoTestReport
+```
+Then, the coverage report will appear in the output directory, inside the `coverage_report` directory.
```

Comment:
I think we should include guidelines what is expected coverage so developers can compare their results